packages/fxa-content-server/tests/functional/avatar.js

```diff
@@ -126,7 +126,7 @@ registerSuite('settings/avatar', {
       );
     },
 
-    'upload a profile image': function () {
+    'upload and then remove profile image': function () {
```

Comment:
Sorta semi related but `go to settings then avatar change` test can probably be removed since this tests it.

Reply:
Yes you are right. Didn't pay attention to the already existing tests. I will remove that test. Thanks!